src/MongoDB.Driver.Core/Core/Events/ConnectionCreatedEvent.cs

```diff
@@ -0,0 +1,87 @@
+﻿/* Copyright 2019-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using MongoDB.Driver.Core.Clusters;
+using MongoDB.Driver.Core.Configuration;
+using MongoDB.Driver.Core.Connections;
+using MongoDB.Driver.Core.Servers;
+
+namespace MongoDB.Driver.Core.Events
+{
+    /// <preliminary/>
+    /// <summary>
+    /// Occurs when a connection is created..
+    /// </summary>
+    /// <remarks>Corresponds to the ConnectionCreatedEvent from the CMAP spec.</remarks>
```

Comment:
None of the other events have a comment like this.

Should we remove this <remarks> line?

Reply:
done